addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -669,7 +669,7 @@ options.Class.include({
     _select: async function (previewMode, widget) {
         await this._super(...arguments);
 
-        if (!widget.$el.closest('[data-no-widget-refresh="true"]').length) {
+        if (this.options.enableWebsite && !widget.$el.closest('[data-no-widget-refresh="true"]').length) {
```

Comment:
I think I need to find a way to remove every include from the website version of web_editor, but I haven't found a way of replacing option instancing yet, so I used this flag in order to only call refresh_public_widget in a website context. Otherwise in mass_mailing we get an infinite loop. (Since there is no PublicWidget to handle the widgets_start_request).